ext/odbc/php_odbc_includes.h

```diff
@@ -220,8 +225,9 @@ typedef struct odbc_result {
 	zend_long longreadlen;
 	int binmode;
 	int fetched;
-	odbc_param_info * param_info;
+	odbc_param_info *param_info;
```

Comment:
I would recommend not to mix white-space and unrelated changes with a big patch. You may commit the obvious changes before.